frontend/src/Editor/ErrorBoundary.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+
+class ErrorBoundary extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { hasError: false };
+  }
+
+  static getDerivedStateFromError(error) {
+    // Update state so the next render will show the fallback UI.
+    return { hasError: true };
+  }
+
+  componentDidCatch(error, errorInfo) {
```

Comment:
I think we should also consider logging in to sentry, if APM for sentry is enabled via env vars.
https://docs.sentry.io/platforms/javascript/guides/react/usage/#capturing-errors

Reply:
Will do this as a separate issue @akshaysasidrn .

Reply:
https://github.com/ToolJet/ToolJet/issues/1619